jmetal-core/src/main/java/org/uma/jmetal/util/experiment/component/GenerateFriedmanTestTables.java

```diff
@@ -236,19 +235,16 @@ public String prepareFileOutputContents(double[] averageRanking) {
   private void writeLatexFile(GenericIndicator<?> indicator, String fileContents) {
     String outputFile = latexDirectoryName +"/FriedmanTest"+indicator.getName()+".tex";
 
-    try {
+    try( FileOutputStream fileOutputStream = new FileOutputStream(outputFile);
+    		DataOutputStream dataOutputStream = new DataOutputStream(fileOutputStream);	) {
       File latexOutput;
       latexOutput = new File(latexDirectoryName);
       if(!latexOutput.exists()){
         latexOutput.mkdirs();
       }
-      FileOutputStream fileOutputStream = new FileOutputStream(outputFile);
-      DataOutputStream dataOutputStream = new DataOutputStream(fileOutputStream);
```

Comment:
It might be more relevant to move the `try` with FOS & DOS here, where they are actually required, and if necessary make another `try` for what is before. I especially think about the relevance of the error messages in the catches.

Reply:
Done in #286.